src/kakarot/interfaces/interfaces.cairo

```diff
@@ -20,6 +20,9 @@ namespace IEth {
 
     func transfer(recipient: felt, amount: Uint256) -> (success: felt) {
     }
+
```

Comment:
maybe we should just rename this `IEth` `IERC20` to be more explicit on the fact that we just use a standard ERC20 and put it directly the whole interface

```
@contract_interface
namespace IERC20:
    func name() -> (name: felt):
    end

    func symbol() -> (symbol: felt):
    end

    func decimals() -> (decimals: felt):
    end

    func totalSupply() -> (totalSupply: Uint256):
    end

    func balanceOf(account: felt) -> (balance: Uint256):
    end

    func allowance(owner: felt, spender: felt) -> (remaining: Uint256):
    end

    func transfer(recipient: felt, amount: Uint256) -> (success: felt):
    end

    func transferFrom(
            sender: felt,
            recipient: felt,
            amount: Uint256
        ) -> (success: felt):
    end

    func approve(spender: felt, amount: Uint256) -> (success: felt):
    end
end
```

Reply:
100% agree,
(We added that as a task for the upcoming Kakarot Workshop ... is that ok?).
We need like ~30 issues for people to work on.
I would almost say the best thing would be to add the Openzeppelin repo as a sub-repo and import the ERC20 and interface from there.... But maybe not the best idea before the cairo1 migration.

Reply:
rn we don't really have a good deps management tool and we've actually closed the issue in favor of waiting for cairo 1 so copypasta party is on atm

Reply:
I'll create simple test to prove that allowance doesn't decrease. 👍

Reply:
@ClementWalter
Actually, we're looking for some easy issues for the upcoming Kakarot workshop.
Can I keep IEth as is for now? Then we'll have the IEth -> IERC20 conversion as a task in the workshop.

Reply:
yes yes for the IEth and the workshop